apps/charterafrica/src/pages/opportunities/[slug].page.js

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 
+import FeaturedPostCard from "@/charterafrica/components/FeaturedPostCard";
```

Comment:
Is this expected to be on Knowledge page or opportunities page cc @kilemensi

Reply:
Yep, should be under `knowledge` folder.

Reply:
@kilemensi @koechkevin This was for the [opportunities page](https://www.figma.com/file/gbnaO1FYPq16V2gAFbLOkl/CfA---EPD-Charter-Africa---Sep-2022?node-id=9108%3A28517&t=Xqi71uwkNMyMyLyH-4). It is very similar to the knowledge page one. I'll customize it as discussed in the morning call